src/seeds.accounts.cpp

```diff
@@ -274,15 +283,28 @@ void accounts::refreward(name account, name new_status) {
   auto uitr = users.find(referrer.value);
   if (uitr != users.end()) {
     auto user_type = uitr->type;
+    auto num_users = is_citizen ? std::distance(citizens.cbegin(),citizens.cend()) : std::distance(residents.cbegin(),residents.cend());
 
```

Comment:
Can't use use distance... we need to keep track of the sizes of these tables in history

so history contract needs a sizes table that keeps track of resident and citizen table sizes - just like we keep track of sizes in other contracts.

Then we need an init method that initializes the size tables with the current values.

Reply:
Done, please review again.